### PR TITLE
reference: update Grafana monitoring best practices

### PR DESCRIPTION
**Pull request by juliezhang1112**
### What is changed, added or deleted? 
- Update the meaning of `Resolution`
- Update the title of tip 5 to make it clearer

### What is the related PR or file link(s)? 
https://pingcap.com/blog-cn/use-grafana-to-monitor-and-analyze-tidb-metrics/

### Which version does your change affect? 
dev, v2.1, v3.0, v3.1

### CODE REVIEW

File: dev/reference/best-practices/grafana-monitor.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ curl http://__tidb_ip__:10080/metrics |grep tidb_executor_statement_total
 ```
 
 ```
-# 可以看到实时 QPS 的数据，区分不同的 type，value 是 counter 类型的累计值（科学计数法）
+# 可以看到实时 QPS 数据，并以不同 type 的 SQL 语句进行了区分，value 是 counter 类型的累计值（科学计数法）。
```
Comment (lilin90) [inline dev/reference/best-practices/grafana-monitor.md]:
```suggestion
# 可以看到实时 QPS 数据，并以不同 type 对 SQL 语句进行了区分，value 是 counter 类型的累计值（科学计数法）。
```

**Comment by lilin90** [comment]
@kissmydb PTAL~ Thanks!

File: dev/reference/best-practices/grafana-monitor.md [hunk 2]
```diff
@@ -10,27 +10,27 @@ category: reference
...
 
 - 如果 Prometheus 需要迁移，无需重启整个 TiDB 集群。调整前，因为组件要调整 push 的目标地址，迁移 Prometheus 需要重启整个集群。
-- 支持部署 2 套独立的 Prometheus（非 HA），防止监控的单点。方法是使用 ansible 用不同的 ip 各执行一次部署命令。
+- 支持部署 2 套独立的 Grafana + Prometheus 的监控平台（非 HA），防止监控的单点。方法是使用 ansible 用不同的 ip 各执行一次部署命令。
 - 去掉了 Pushgateway 这个单点组件。
 
 ## 监控数据的来源与展示
 
-TiDB 的 3 个核心组件（TiDB server、TiKV server、PD server）可以通过 http 接口来获取 metric 数据。这些指标均是从程序代码中统计上传的，端口如下：
+TiDB 的 3 个核心组件（TiDB server、TiKV server 和 PD server）可以通过 HTTP 接口来获取 metric 数据。这些 metric 均是从程序代码中上传的，端口如下：
```
Comment (kissmydb) [inline dev/reference/best-practices/grafana-monitor.md]:
```suggestion
TiDB 的 3 个核心组件（TiDB server、TiKV server 和 PD server）可以通过 HTTP 接口来获取 metric 数据。这些 metric 均是从程序代码中上传的，默认端口如下：
```

Comment (kissmydb) [inline dev/reference/best-practices/grafana-monitor.md]:
```suggestion
TiDB 的 3 个核心组件（TiDB server、TiKV server 和 PD server）可以通过 HTTP 接口来获取 metric 数据。这些 metric 均是从程序代码中上传的，默认端口如下：
```

**Comment by juliezhang1112** [comment]
@kissmydb PTAL again~ Thanks!

**Pull request merged**